_data/projects/ushahidi.yml

```diff
@@ -0,0 +1,13 @@
+name: Ushahidi
+desc: Ushahidi is an open source web application for information collection, visualization and interactive mapping. It helps you to collect info from: SMS, Twitter, Email.
```

Comment:
@rowasc I guess `:` in your `desc` is causing issues as `:` is used for mapping in .yml

Reply:
oops, thank you ! changed